### PR TITLE
Performance investigation upgrade Python to 3.9

### PR DESCRIPTION
**Pull request by JamesGardiner**
### What is the context of this PR?
Performance investigation 0017 - Upgrade Python to 3.9

### How to review 
Check the new performance investigation docs

### CODE REVIEW

File: doc/performance-investigations/0017-upgrade-python-3.9/summary.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Title
```
Comment (ajmaddaford) [inline doc/performance-investigations/0017-upgrade-python-3.9/summary.md:1]: The title could be a bit more descriptive 😉

File: doc/performance-investigations/0017-upgrade-python-3.9/summary.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+| Option | Value |
+|--------|-------|
+| Requests file | census_household_gb_eng.json |
+| Run time | 60m |
+| User wait time minimum | 1 |
+| User wait time maximum | 2 |
+| Clients | 100 |
+| Hatch rate | 100 |
+| Number of workersi | 7 |
```
Comment (MebinAbraham) [inline doc/performance-investigations/0017-upgrade-python-3.9/summary.md:15]:
```suggestion
| Number of workers | 7 |
```

File: doc/performance-investigations/0017-upgrade-python-3.9/summary.md (hunk 1 above)
Comment (JamesGardiner) [reply doc/performance-investigations/0017-upgrade-python-3.9/summary.md:1]: 🤦

File: doc/performance-investigations/0017-upgrade-python-3.9/summary.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Results
+
+Results based on 99th percentile timings.
+
+| Metric              | Baseline | Investigation |
+| ------------------- | -------- | ------------- |
+| Questionnaire GETs  | 198ms    | 154ms         |
+| Questionnaire POSTs | 299ms    | 250ms         |
+| All requests        | 247ms    | 202ms         |
```
Comment (MebinAbraham) [inline doc/performance-investigations/0017-upgrade-python-3.9/summary.md:26]: Can you use the outputs from get_summary script. Would be good to include the total requests served.

Comment (JamesGardiner) [reply doc/performance-investigations/0017-upgrade-python-3.9/summary.md:26]: Yeah sure thing

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**